randomActivityGen.py

```diff
@@ -91,6 +92,88 @@ def setup_city_gates(net: sumolib.net.Net, stats: ET.ElementTree, gate_count: in
         })
 
 
+def find_school_edges(net: sumolib.net.Net, num_schools):
+    edges = net.getEdges()
+
+    # Sort all edges based on their avg coord
+    edges.sort(key=lambda x: np.mean(get_edge_pair_centroid(x.getShape())))
+
+    # Split edges into n districts, with n being number of schools
+    district_size = int(np.ceil(len(edges) / num_schools))
+    districts = [edges[x:x + district_size] for x in range(0, len(edges), district_size)]
+
+    school_edges = []
+
+    def noise(edge):
+        x, y = get_edge_pair_centroid(edge.getShape())
+        return get_perlin_noise(x, y, POPULATION_BASE)
+
+    # Pick out the one edge with highest perlin noise from each district and return these to later place school on
+    for district in districts:
+        district.sort(key=noise)
+        school_edges.append(district[-1])
+
+    return school_edges
+
+
+def setup_schools(net: sumolib.net.Net, stats: ET.ElementTree, school_count: int or None):
+    # Voodoo parameter, seems to be about the value for a couple of danish cities.
+    # In general one high school, per 5000-7000 inhabitant in a city
+    inhabitants_per_school = 5000
+
+    school_opening_earliest = 7 * 3600
+    school_opening_latest = 10 * 3600
+    school_closing_earliest = 13 * 3600
+    school_closing_latest = 17 * 3600
+    stepsize = int(0.25 * 3600)
```

Comment:
Maybe these should be parameterised with these values as defaults? Also the ranges for `begin_age` and `end_age`.

Reply:
I also thought about doing this, will add.

Reply:
Added in 3520bb1a7dd3b7adeffce44f1dd24f808253abec